NEWS.rst

```diff
@@ -6,6 +6,7 @@ Internal changes
 
 * 23799: Added custom UUID url converter, stringifying UUID parameters in
   order to standardise our use of UUIDs internally.
+* #24797 Integration_data added to employee and org.unit
```

Comment:
Strengt taget er dette her vel almindelig tale, og så vil det være mest naturligt at skrive ("integration data") i to ord uden underscore (undskyld, jeg er så vanskelig! :-))

Reply:
Er delt!